controllers/reivewsController.ts

```diff
@@ -0,0 +1,46 @@
+import { RequestHandler } from "express";
+
+import { PrismaClient, Prisma } from "@prisma/client";
+const prisma = new PrismaClient();
+
+export const approveReview: RequestHandler = async (req, res) => {
+    const reviewId: number = Number(req.params.reviewId);
+
+    // TODO? Turn not found checker to middleware
```

Comment:
sorry what does this mean?

Reply:
Small not to myself sana that a not found middleware can be built for requests that require finding a row a table then proceeding with some logic. 

Is that okay to put it on the codebase or I should remove it and just make the note on my personal todo list?

Reply:
Yea have it as //TODO, it's fine :)

Reply:
Got it! Thank you